e3sm_diags/derivations/acme.py

```diff
@@ -133,6 +133,14 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = var / (1.0 + var) * 1000.0  # convert to g/kg
```

Comment:
@tangq would you check if the unit conversion is correct?

Reply:
@chengzhuzhang , you will need the molecular weights of H2O (18 g/mol) and dry air (28.97 g/mol) to convert mol/mol to g/kg.
I notice that some typos (k/kg) of the units on figures like: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/zhang40/relative_diff_try2/viewer/zonal_mean_2d/merra2/h2olnz-global-merra2/ann.html